api/src/main/java/org/openmrs/api/context/ServiceContext.java

```diff
@@ -143,10 +143,8 @@ public static ServiceContext getInstance() {
 	 */
 	public static void destroyInstance() {
 		if (ServiceContextHolder.instance != null && ServiceContextHolder.instance.services != null) {
-			if (log.isDebugEnabled()) {
 				for (Map.Entry<Class, Object> entry : ServiceContextHolder.instance.services.entrySet()) {
-					log.debug("Service - " + entry.getKey().getName() + ":" + entry.getValue());
-				}
+					log.debug("Service - {} : {}", entry.getKey().getName(), entry.getValue());
```

Comment:
```suggestion
				log.debug("Service - {} : {}", entry.getKey().getName(), entry.getValue());
```

the log.debug is inside the for loop, so we need to make sure the indentation is right. otherwise its hard to read. so it should stay as

```java
for () {
    log.debug()
}
```

please use tabs (writing this on my phone where I don't know how to insert tabs :)